lxde-base/lxdm/lxdm-0.4.1-r10.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="2"
```

Comment:
I'm sorry, but I don't know much about EAPI changes and everything related to that. Therefore someone else has to take care of this.